src/spox/_inline.py

```diff
@@ -13,6 +15,48 @@
 from . import _value_prop
 
 
+def rename_in_graph(
+    graph_: onnx.GraphProto,
+    rename: Callable[[str], str],
+    *,
+    rename_node: Optional[Callable[[str], str]] = None,
+    rename_op_type: Optional[Callable[[str], str]] = None,
+) -> onnx.GraphProto:
+    graph = onnx.GraphProto()
+    graph.CopyFrom(graph_)
+
+    for p in itertools.chain(graph.input, graph.initializer):
+        p.name = rename(p.name)
+    for si in graph.sparse_initializer:
+        si.values.name = rename(si.values.name)
+        si.indices.name = rename(si.indices.name)
+
+    for nd in graph.node:
+        if nd.name and rename_node is not None:
```

Comment:
Are nodes really allowed to have a missing `name` / `op_type`?

Reply:
https://github.com/onnx/onnx/blob/main/docs/IR.md#nodes

> name | string | An optional name of the node, used for diagnostic purposes only.

I think it's often left empty/unset in practice, though Spox does build the nodes with its internal naming.

You're right about `op_type`, though. FYI, replacing `op_type` is there for future-proofing as inlining models with functions might require renaming some local functions.